### PR TITLE
Updated populate task #1589

### PR DESCRIPTION
**Pull request by michaelDpierce**

### CODE REVIEW

File: lib/tasks/sample_data.rake [hunk 1]
```diff
@@ -185,17 +144,12 @@ namespace :db do
     10.times do |n|
       print '...'
 
-      Certification.create!(certification_name_id: 1,
+      Certification.create!(certification_name_id: CertificationName.first.id,
                             user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: all user_id's need to be safe as well. After creating users and I would create a sample array:
user_ids_array = []
User.all.each do |user|
user_ids_array.push user.id
end

then later on user_id: user_ids:array.sample for your Certs, HelpDesk, etc.

File: lib/tasks/sample_data.rake [hunk 2]
```diff
@@ -259,11 +213,45 @@ namespace :db do
                              si_index: [-1, -0.5, -0.3, 0, 0.3, 0.5, 1].sample,
                              si_status: si_status.sample,
                              si_recommendation: si_recommendation.sample,
-                             pool_id: Pool.take(3).map(&:id).sample,
-                             location_id: Location.take(2).map(&:id).sample,
+                             pool_id: Pool.first.id,
+                             location_id: Location.first.id,
                              user_id: [1, 2, 3, 4, 5].sample)
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 3]
```diff
@@ -185,17 +144,12 @@ namespace :db do
...
                             issue_date: Date.today + rand(365).day)
 
-      Certification.create!(certification_name_id: 2,
-                            user_id: n + 1,
-                            expiration_date: Date.today + rand(365).day,
-                            issue_date: Date.today + rand(365).day)
-
-      Certification.create!(certification_name_id: 3,
+      Certification.create!(certification_name_id: CertificationName.last.id,
                             user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake (hunk 1 above)
Comment (michaelDpierce) [inline lib/tasks/sample_data.rake]: Okay good deal.

File: lib/tasks/sample_data.rake [hunk 4]
```diff
@@ -259,11 +213,45 @@ namespace :db do
...
+                pool_temp: [81, 82, 83, 84, 85, 86].sample,
+                air_temp: [75, 76, 77, 78, 79, 80].sample,
+                water_clarity: water_clarity.sample,
+                time_stamp: Time.now,
+                si_index: [-1, -0.5, -0.3, 0, 0.3, 0.5, 1].sample,
+                si_status: si_status.sample,
+                si_recommendation: si_recommendation.sample,
+                pool_id: Pool.last.id,
+                location_id: Location.last.id,
+                user_id: [1, 2, 3, 4, 5].sample)
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 5]
```diff
@@ -274,8 +262,8 @@ namespace :db do
       HelpDesk.create!(name: name.sample,
                        urgency: %w(Low Medium High).sample,
                        issue_status: true,
-                       location_id: 1,
-                       user_id: 1,
+                       location_id: Location.first.id,
+                       user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 6]
```diff
@@ -274,8 +262,8 @@ namespace :db do
...
                        closed_user_id: n + 2
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 7]
```diff
@@ -285,7 +273,7 @@ namespace :db do
     10.times do |n|
       print '.'
       ShiftReport.create!(user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 8]
```diff
@@ -336,13 +324,13 @@ namespace :db do
     TimeOffRequest.create!(user_id: 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 9]
```diff
@@ -350,23 +338,23 @@ namespace :db do
...
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 10]
```diff
@@ -350,23 +338,23 @@ namespace :db do
...
-                           reason: 'Blah Blah Blah',
+                           reason: 'Family Vacation',
                            approved: false,
                            active: true,
-                           location_id: rand(3) + 1
+                           location_id: [Location.first.id, Location.last.id].sample
                           )
 
     print '.'
     TimeOffRequest.create!(user_id: 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 11]
```diff
@@ -376,7 +364,7 @@ namespace :db do
       print '.'
       SlideInspection.create!(slide_id: Slide.first.id,
                               user_id: rand(80) + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 12]
```diff
@@ -384,7 +372,7 @@ namespace :db do
       print '.'
       SlideInspection.create!(slide_id: Slide.last.id,
                               user_id: rand(80) + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 13]
```diff
@@ -392,7 +380,7 @@ namespace :db do
       print '.'
       SlideInspection.create!(slide_id: Slide.first.id,
                               user_id: rand(80) + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 14]
```diff
@@ -401,8 +389,8 @@ namespace :db do
     50.times do |n|
       print '.'
       Shift.create!(user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

File: lib/tasks/sample_data.rake [hunk 15]
```diff
@@ -473,14 +461,14 @@ namespace :db do
...
+                    position_id: [Position.first.id, Position.last.id].sample,
                     start_time: Time.now + rand(3).hours - 144.hours,
                     end_time: Time.now + rand(5).hours - 140.hours)
     end
 
     puts "\n\nAvailabilities"
-    50.times do |n|
+    10.times do |n|
       print '.'
       Availability.create!(user_id: n + 1,
```
Comment (duffcodester) [inline lib/tasks/sample_data.rake]: make safe

**Pull request merged**